.github/workflows/tests.yml

```diff
@@ -98,3 +98,177 @@ jobs:
     - name: Test with pytest
       run: |
         pytest -v ./tests/sealapi
+  
+  TenSEAL_WIN:
+
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+      
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements_dev.txt
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 --config=.flake8 --exclude=tenseal/sealapi,tests/sealapi .
+        black --check --exclude "sealapi" tenseal/ tests/
+        # exit-zero treats all errors as warnings.
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=100 --statistics
+    - name: Lint with clang-format
```

Comment:
I don't think we need to duplicate the linting logic to all platforms, Linux is enough. It would fail anyway

Reply:
ok i will make changes